docker-compose.yml

```diff
@@ -133,6 +133,12 @@ services:
     volumes:
     - ./oaipmh/fiz-oai-backend.properties:/usr/local/tomcat/conf/fiz-oai-backend.properties:ro
     - backend-logs:/usr/local/tomcat/logs
+    labels:
+      - traefik.http.routers.oai-provider.rule=Host(`oai.portal.mardi4nfdi.${TLD}`)
+      - traefik.http.routers.oai-provider.entrypoints=websecure
+      - traefik.http.routers.oai-provider.tls.certResolver=le
+      - traefik.http.middlewares.test-auth.basicauth.users=bWFyZGktdGVzdC11c2VyOnRlc3Q=
```

Comment:
the username is mising

Reply:
the username was encoded with the original form "user:password" and processed with https://mixedanalytics.com/tools/basic-authentication-generator/

Reply:
This should happen on the client side. For the server side, we need the information that usually would go to .htaccess You can generate it here http://aspirine.org/htpasswd_en.html keep the password private as this repo is public.

Reply:
The build fails

https://github.com/MaRDI4NFDI/portal-compose/actions/runs/7492851367/job/20397278707?pr=459#step:4:21

, as the password is not properly escaped, see https://doc.traefik.io/traefik/middlewares/http/basicauth/